virttest/qemu_storage.py

```diff
@@ -187,6 +187,10 @@ def filename_to_file_opts(filename):
 
                 if matches['user'] is not None:
                     file_opts['username'] = matches['user']
+    # FIXME: vdpa is a customized protocol instead of a standard protocol.
+    elif filename.startswith("vdpa:"):
+        file_opts = {'driver': 'virtio-blk-vhost-vdpa',
+                     'path': filename.removeprefix("vdpa://")}
```

Comment:
The `removeprefix` method is supported starting from python 3.9, however, the VT framework is required to be compatible with python 3.7 and 3.8. We have to find another approach instead.

https://docs.python.org/3.10/library/stdtypes.html?highlight=removeprefix#str.removeprefix